packages/xo-server/src/xapi/index.mjs

```diff
@@ -962,10 +962,10 @@ export default class Xapi extends XapiBase {
   @synchronized()
   _callInstallationPlugin(hostRef, vdi) {
     return this.call('host.call_plugin', hostRef, 'install-supp-pack', 'install', { vdi }).catch(error => {
-      if (error.code !== 'XENAPI_PLUGIN_FAILURE') {
-        log.warn('_callInstallationPlugin', { error })
+      if (error.code !== 'XENAPI_PLUGIN_FAILURE' || !error.params?.[2]?.includes?.('UPDATE_ALREADY_APPLIED')) {
```

Comment:
I just made the test to ignore the error more precise so that other errors would be correctly thrown. And I moved `log.warn` since I think it was mistake: we want to show a warning when we're about to ignore the error, not the other way around.